packages/CalendarCarousel/index.tsx

```diff
@@ -1,19 +1,269 @@
-import { StyleSheet, Text, View } from 'react-native';
+import {
+  FlatList,
+  SafeAreaView,
+  StyleSheet,
+  Text,
+  TextStyle,
+  TouchableOpacity,
+  View,
+  ViewStyle,
+} from 'react-native';
+import React, { ReactElement, useState } from 'react';
 
-import React from 'react';
+interface Style {
+  container: ViewStyle;
+  headerFlex: ViewStyle;
+  beforeArrowText: TextStyle;
+  titleFlex: ViewStyle;
+  titleText: TextStyle;
+  yearText: TextStyle;
+  nextArrowText: TextStyle;
+  weekdayFlex: ViewStyle;
+  weekdayText: TextStyle;
+  calendarContainerSize: ViewStyle;
+  noPressView: ViewStyle;
```

Comment:
What is `noPress`?

Reply:
haha it means when not pressed. I changed it to notActive !